src/main/java/at/hannibal2/skyhanni/api/GetFromSackAPI.kt

```diff
@@ -222,12 +223,25 @@ object GetFromSackAPI {
     }
 
     @SubscribeEvent
-    fun onRepoReload(event: RepositoryReloadEvent) {
-        sackListInternalNames = event.getConstant<SacksJson>("Sacks").sackItems.toSet()
+    fun onNeuRepoReload(event: NeuRepositoryReloadEvent) {
+        val data = event.getConstant("sacks") ?: ErrorManager.skyHanniError("NEU sacks data is null.")
+        try {
+            val sacksData = ConfigManager.gson.fromJson<NeuSacksJson>(data).sacks
+            val uniqueSackItems = mutableSetOf<NEUInternalName>()
+
+            sacksData.values.forEach { sackInfo ->
+                sackInfo.contents.forEach { content ->
+                    uniqueSackItems.add(content)
+                }
+            }
+
+            sackListNames = uniqueSackItems.map { it.itemNameWithoutColor.uppercase() }.toSet()
 
-        DelayedRun.runNextTick {
-            sackListNames = sackListInternalNames.map { it.asInternalName().itemNameWithoutColor.uppercase() }.toSet()
+        } catch (e: Exception) {
+            ErrorManager.logErrorWithData(
+                e, "Error getting NEU sacks data, make sure your neu is updated.",
```

Comment:
maybe tell "your neu repo" instead. and as long as we depend on neu, we could even run `Utils.showOutdatedRepoNotification()`